### ISSUE TITLE
MSAA disabled when replaying a trace

### ISSUE THREAD

**Issue by AnyOldName3** [apitrace/apitrace#690]
Tested on Windows with OpenMW (https://gitlab.com/OpenMW/openmw) which uses OpenGL 2.1 and renders to the main window's framebuffer.

When capturing a trace, MSAA works correctly, but when replaying it, it doesn't. It *looks* to me like the WGL calls at the start of the trace are requesting the pixel format number to be one with MSAA, but the pixel format number returned and used is one without. I'm not sure whether the WGL calls in the trace are what OpenMW calls directly or whether they've been adjusted by APITrace as we use SDL2 to abstract this away.

**Comment by jrfonseca** [comment]
Yes, this is a know limitation of glretrace.  It would be fine to fix it eventually, but it implies a fair amount of work, due to the fact that WGL/GLX/CGL calls are never replayed verbatim, but go through an emulation layer to allow replaying on one OS a trace captured on a different OS.

**Comment by jrfonseca** [comment]
@gerddie  , FYI, this is the issue I mentioned on https://github.com/apitrace/apitrace/pull/700#issuecomment-773411284
